tests/unit/mock/config/parser/base/mikrotik_routeros/routeros_full_received.py

```diff
@@ -0,0 +1,151 @@
+    ConfigLine(
+        config_line="/ip ipsec policy add disabled=yes dst-address=169.254.1.1/32 peer=google-vpc-peer src-address=169.254.1.2/32 tunnel=yes",
+        parents=(),
+    ),
+    ConfigLine(
+        config_line="/ip ipsec policy add disabled=yes dst-address=10.150.0.0/20 peer=google-vpc-peer src-address=0.0.0.0/0 tunnel=yes",
+        parents=(),
+    ),
+    ConfigLine(config_line="/ip service set telnet disabled=yes", parents=()),
+    ConfigLine(config_line="/ip service set ftp disabled=yes", parents=()),
+    ConfigLine(config_line="/ip service set www disabled=yes", parents=()),
+    ConfigLine(config_line="/ip service set api address=10.1.15.5/32,10.120.16.0/20,172.16.11.0/24", parents=()),
+    ConfigLine(config_line="/ip service set api-ssl disabled=yes", parents=()),
+    ConfigLine(config_line="/routing bgp network add network=55.106.77.11/32 synchronize=no", parents=()),
+    ConfigLine(config_line="/routing bgp network add network=55.106.77.12/32 synchronize=no", parents=()),
+    ConfigLine(
+        config_line="/routing bgp peer add in-filter=ibgp-allow-default-in name=CSW out-filter=ibgp-no-default-out remote-address=10.127.1.10 remote-as=12345 update-source=loopback",
+        parents=(),
+    ),
+    ConfigLine(
+        config_line="/routing filter add action=discard chain=ibgp-no-default-out prefix=10.127.1.0/24", parents=()
+    ),
+    ConfigLine(
+        config_line="/routing filter add action=accept chain=ibgp-no-default-out prefix=10.64.0.0/10 prefix-length=22-32",
+        parents=(),
+    ),
+    ConfigLine(
+        config_line="/routing ospf interface add dead-interval=8s hello-interval=2s interface=vlan3049-Transit-to-85Presidential network-type=point-to-point",
+        parents=(),
+    ),
+    ConfigLine(
+        config_line="/routing ospf interface add cost=11 dead-interval=8s hello-interval=2s interface=vlan3166-Transit-HalseyCore-Agg1-to-Indigo-5 network-type=point-to-point",
+        parents=(),
+    ),
+    ConfigLine(config_line="/routing ospf network add area=backbone network=10.126.0.16/29", parents=()),
+    ConfigLine(config_line="/routing ospf network add area=backbone network=10.127.1.3/32", parents=()),
+    ConfigLine(config_line="/snmp set enabled=yes trap-community=somestring", parents=()),
+    ConfigLine(config_line="/system clock set time-zone-name=America/New_York", parents=()),
+    ConfigLine(config_line="/system identity set name=ag1.123site.nwk.nj", parents=()),
+    ConfigLine(config_line="/system logging add action=remote topics=error", parents=()),
+    ConfigLine(config_line="/system logging add action=remote topics=info", parents=()),
+    ConfigLine(
+        config_line='/system note set note="This is a \\\\\\"System Note\\\\\\" for a Mikrotik router.\\\\n\\\\\\',
+        parents=(),
+    ),
+    ConfigLine(
+        config_line='    \\nIt includes double quotes (\\\\\\") and special characters such as:\\\\n\\\\\\',
```

Comment:
In my mind, this line and the below should all be a single line without a parent. As an example I would not see the other lines as children of this parent.

look at: https://github.com/networktocode/netutils/blob/1312c08d58a74ebb63f36ea35d03bec7747628d0/tests/unit/mock/config/parser/base/cisco_ios/ios_full_sent.txt#L288 and https://github.com/networktocode/netutils/blob/1312c08d58a74ebb63f36ea35d03bec7747628d0/tests/unit/mock/config/parser/base/cisco_ios/ios_full_received.py#LL360C34-L360C86

Reply:
Agreed, I'd expect this to be one single line.

Reply:
Right on, the more I looked at this, it makes sense to treat this like a banner since it does allow to display upon logging into the device. Made some changes, lmk what ya'll think.  Single line.